spec/integration/subset_spec.rb

```diff
@@ -36,4 +37,25 @@
     subset1 = TTFunk::Subset.for(font, :unicode)
     expect { subset1.encode }.to_not raise_error
   end
+
+  it 'calculates correct search_range, entry_selector and range_shift values' do
+    font = TTFunk::File.open test_font('DejaVuSans')
+
+    subset = TTFunk::Subset.for(font, :unicode)
+    subset.use(97)
+    subset_io = StringIO.new(subset.encode)
+
+    scaler_type, table_count = subset_io.read(6).unpack('Nn')
+    search_range, entry_selector, range_shift = subset_io.read(6).unpack('nnn')
```

Comment:
Isn't there a way to use TTFunk to get this out of generated subset font?

Reply:
@pointlessone Currently there is no way; TTFunk [parses out the scaler type and table count](https://github.com/prawnpdf/ttfunk/blob/635205f9b633e5a432338d401b087fc8e5987ed9/lib/ttfunk/directory.rb#L8), but discards/ignores they bytes that represent search range, entry selector, and range shift.

I could change `TTFunk::Directory` to store these these three values and make them available as attribute readers. Does that sound good?

Reply:
No, don't bother. Apparently we don't use them for anything so no need to store them.